drivers/enc28j60/enc28j60.c

```diff
@@ -260,6 +286,12 @@ static int nd_send(netdev_t *netdev, const iolist_t *iolist)
     cmd_w_addr(dev, ADDR_TX_END, cmd_r_addr(dev, ADDR_WRITE_PTR) - 1);
     /* trigger the send process */
     cmd_bfs(dev, REG_ECON1, -1, ECON1_TXRTS);
+    /* set last transmission time for timeout handling */
+    dev->tx_time = xtimer_now_usec();
+
+#ifdef MODULE_NETSTATS_L2
+    netdev->stats.tx_bytes += c;
```

Comment:
IMHO no need to do this twice 😉 see below

Reply:
@sming Sure, I don't know how this could happen :-(